docs/faq.md

```diff
@@ -16,12 +16,12 @@ If you don’t see your question here, feel free to drop by `#docker-compose` on
 freenode IRC and ask the community.
 
 
-## Can I control service startup order?
+### Can I control service startup order?
```

Comment:
I agree. I raised this issue as well on a related change. I think we should address this with a change to css instead of changing the markdown.